drivers/metadata/metadata.go

```diff
@@ -163,7 +163,7 @@ type Writer interface {
 }
 
 type CatalogSet struct {
-	resultSet
+	*resultSet
```

Comment:
Is this still required or is it a leftover after removing `AddResult`?

Reply:
no required.